prow/istio-pilot-e2e.sh

```diff
@@ -48,12 +48,13 @@ fi
 
 source "${ROOT}/prow/cluster_lib.sh"
 
-trap delete_cluster EXIT
-create_cluster 'e2e-pilot'
+#trap delete_cluster EXIT
+#create_cluster 'e2e-pilot'
 
-ln -sf "${HOME}/.kube/config" ${ROOT}/pilot/pkg/kube/config
+KUBECONFIG=${ROOT}/pilot/pkg/kube/config
```

Comment:
prow job fails, it cannot find it:
	error	kubernetes configuration file "/home/bootstrap/go/src/istio.io/istio/pilot/pkg/kube/config" does not exist